### DISCUSSION TITLE
Has Apple shown any interest in CLAP for Logic Pro X?

### DISCUSSION THREAD

**Discussion by NickLarsenNZ** [free-audio/clap#184] [General]
I see a number of DAWs evaluating CLAP, and I'd like to use a DAW that supports CLAP.

I currently use Apple Logic Pro X, and my main gripe from a user perspective is that it doesn't support VSTs (sometimes there just isn't an AU equivalent). Now that CLAP has come along, I can imagine many of the VST plugins will also come as CLAP, so I'd likely move to a DAW that supports CLAP and be done with the other formats.

So I wonder, is Apple coming to the party, or will they keep up the walled garden (AU plugins).

From a developer perspective, IMO, AU plugins are horrible to develop. And FWIW, I would much prefer develop plugins in Rust, so it's nice to see the Rust bindings (rust-sys) there 👍.

**Comment by abique** [comment]
Hi,
I'm not aware of Apple Logic Pro's roadmap.
Did you try to ask them directly?
Cheers,
Alex

**Comment by baconpaul** [comment]
Our strategy also has been to develop wrappers which load CLAPs inside other formats. We have a VST3 version of this working (so it is a VST3 which proxies to a CLAP) and have started on an AU.

My music environments are Logic, Bespoke, Rack and Bitwig so I definitely would need the wrappers for CLAP usage if there were pure -play claps. 

For rust, I think you may be best to use nih-plug by the way! It's pretty awesome :)
